lib/lita-lean_tc.rb

```diff
@@ -62,6 +66,15 @@ def build_finished(payload)
           create_confirmed
         end
       end
+      # Returns a list of all cards on the Feature request board
+      #
+      def list_feature_request(response)
```

Comment:
Yep, some of it is duplicated. I'll try to extract the logic in the `to_msg` method so it can be used in the `list_feature_request` method. :)